src/app/routes/utils/pageTypes.js

```diff
@@ -1,7 +1,9 @@
 export const ARTICLE_PAGE = 'article';
```

Comment:
Do we need to give this unit test coverage as we're going to be using these throughout the codebase? Or because they're constants they should be covered by unit tests for whatever is importing and using them?

Reply:
What would you test here, it's just a list of consts?